daemon/internal/newrelic/integration/test.go

```diff
@@ -456,6 +458,122 @@ func (t *Test) compareSpanEventsLike(harvest *newrelic.Harvest) {
 	}
 }
 
+func (t *Test) compareMetricsExist(harvest *newrelic.Harvest) {
+	for _, spec := range strings.Split(strings.TrimSpace(string(t.subEnvVars(t.metricsExist))), "\n") {
+		var err error
+		var count int64
+
+		fields := strings.Split(spec, ",")
+		name := fields[0]
+		name = strings.TrimSpace(name)
+
+		// -1 means just test for existence not an exact count
+		count = -1
```

Comment:
since count is set to -1 here already, do we need to set it additionally [here](https://github.com/newrelic/newrelic-php-agent/pull/921/files#diff-c7f1af7665ab0865f15e79eeba7e73a7295fe66e0ff9205b9781c7b0d7c75faeR478) and [here](https://github.com/newrelic/newrelic-php-agent/pull/921/files#diff-c7f1af7665ab0865f15e79eeba7e73a7295fe66e0ff9205b9781c7b0d7c75faeR486)?

Reply:
I went back and forth on this - I felt it might help readability to have the alternate condition in those two cases balanced by setting a default value.   The code will work fine without those however and I imagine the optimizer removes that code anyways.